src/lib/pubkey/ec_group/info.txt

```diff
@@ -11,12 +11,14 @@ brief -> "Wrapper for elliptic curve groups"
 <requires>
 asn1
 numbertheory
+pcurves
```

Comment:
Is that really required? If `pcurves` is not compiled, I can still fall back to the bigint implementation also for the pcurves-supported curves, no?

Or do we want to model "no pcurves at compile time" by disabling all `pcurves_*` submodules?

Reply:
Not really required in that if pcurves wasn't available at all we could just fall back to the generic code without issue. I'm just trying to avoid too much combinatoric explosion (more than is already in place anyway ...) with yet another module / yet more ifdef. IDK. I guess there are probably users who would prefer just using the generic code for size reasons.

Reply:
#4212 allows the user to disable all curves by disabling `pcurves_impl` which removes all of the implementations while still leaving the top level interfaces in place.